README.md

```diff
@@ -28,18 +28,20 @@ Add this to your application.js after page is ready:
 
     $('.initialjs-avatar').initial();
 
-And finally, in your views, tag your *<img/>* class with `initialjs-avatar` class. You can use *data-attributes* as described in [initial.js](https://github.com/mozilla/metrics-graphics). Or you can use the helper below.
+And finally, in your views, tag your *<img/>* class with `initialjs-avatar` class. You can use *data-attributes* as described in [initial.js](https://github.com/judesfernando/initial.js). Or you can use the helper below.
 
 Enjoy.
 
 ## ActionView Helper
 
-For extra simplicity, you can use this helper in your view to generate your avatar image from your model. In the following examples we'll use User, but any model responding to `:name` will do. You can learn the options available and their defaults directly from the examples:
+For extra simplicity, you can use this helper in your view to generate your avatar image from your model. In the following examples we'll use User, but any model responding to `:full_name` will do. You can learn the options available and their defaults directly from the examples:
```

Comment:
In the second PR, I'd like to support old behavior as well. Something like 

name = avatarable.responds_to?(:name) ? avatarable.name : avatarable